src/main/io/gps.c

```diff
@@ -494,6 +494,15 @@ void gpsUpdate(timeUs_t currentTimeUs)
             gpsNewData(serialRead(gpsPort));
     }
 
+    // GPS data received via MSP
+    if (GPS_update & 2) {
+        gpsSetState(GPS_RECEIVING_DATA);
+        gpsData.lastMessage = millis();
+        sensorsSet(SENSOR_GPS);
+        onGpsNewData();
+        GPS_update = 0;
```

Comment:
This will interfere with bit 0 toggling .. probably not important, but `GPS_update &= ~2;` may be safer

Reply:
Agreed.